packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/WfoSidebar.tsx

```diff
@@ -86,7 +84,6 @@ export const WfoSidebar: FC<WfoSidebarProps> = ({ overrideMenuItems }) => {
         {
             name: t('metadata'),
             id: '5',
-            href: PATH_METADATA,
             onClick: () => {
                 router.push(PATH_METADATA);
```

Comment:
meta data is missing the e.preventDefault -- that might be the cause of the reloading

Reply:
I have readded the hrefs to the menu items, I noticed they are used in the policyCheck. Using the onClick to handle the navigation here discards the middleClick functionality that we have discussed so I have replaced it with a renderItem  function that renders a Link item

Reply:
Adding the preventDefault, just like in other menu items, would avoid this problem.